static/templates/settings/account_settings.hbs

```diff
@@ -160,6 +160,7 @@
               delete_text = (t "Delete profile picture")
               is_editable_by_current_user = user_can_change_avatar
               image = page_params.avatar_url_medium}}
+            <div id ="avatar_image_file_input_error" class="text-error"></div>
```

Comment:
I don't think we want to add a separate error feature outside the `image_upload_widget`; we want to just make `image_upload_widget` error handling work correctly.